src/web/templates/admin/event/tab.html

```diff
@@ -52,7 +52,75 @@
         <tbody>
             <tr>
                 <th scope="row" class="text-nowrap">{{ _('Unique ID') }}</th>
-                <td class="text-nowrap">{{ admin_event.uniq_id }}</td>
+                <td class="text-nowrap">
+                    <span id="uniq-id-field">
+                        {{ admin_event.uniq_id }}
+                        <span
+                            {{ macros.tooltip_attributes(
+                                'info',
+                                _(
+                                    'Edit the unique ID of the event, used for URLs and file exports. '
+                                    'This will cause an error on any opened screen.'
+                                ),
+                                'right'
+                            ) }}
+                        >
+                            <button
+                                id="uniq-id-update-button"
+                                data-testid="uniq-id-update-button"
+                                type="button"
+                                class="btn btn-sm text-primary text-nowrap"
+                            >
+                                <i class="bi-pencil"></i>
+                            </button>
+                        </span>
+                    </span>
+                    <div
+                        id="uniq-id-update-form-container"
+                        class="col-12 col-md-6"
```

Comment:
```suggestion
                        class="col-12"
```
<img width="1156" height="532" alt="image" src="https://github.com/user-attachments/assets/c1e57ebd-9b0f-4d2c-8839-d09a8902746f" />